From a893a953217ad8c8041de64049e133c12eb5ee54 Mon Sep 17 00:00:00 2001
From: Viktor Szakats <vszakats@users.noreply.github.com>
Date: Tue, 21 Feb 2017 23:01:37 -0500
Subject: [PATCH] urldata: include curl_sspi.h when Windows SSPI is enabled

f77dabe broke builds in Windows using Windows SSPI but not Windows SSL.

Bug: https://github.com/curl/curl/issues/1276
Reported-by: jveazey@users.noreply.github.com
(cherry picked from commit f4739f639fc481a7bec7da73041dfcb3d26ed220)

diff --git a/lib/urldata.h b/lib/urldata.h
index 648b3e8..7f87913 100644
--- a/lib/urldata.h
+++ b/lib/urldata.h
@@ -136,8 +136,10 @@
 #undef realloc
 #endif /* USE_AXTLS */
 
-#ifdef USE_SCHANNEL
+#if defined(USE_SCHANNEL) || defined(USE_WINDOWS_SSPI)
 #include "curl_sspi.h"
+#endif
+#ifdef USE_SCHANNEL
 #include <schnlsp.h>
 #include <schannel.h>
 #endif
-- 
2.9.3

